.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+language: python
+python:
+- '3.6'
+install: pip install tox-travis
+script: tox
```

Comment:
All of the above was copied and pasted from https://github.com/tox-dev/tox-travis.
My goal was to make it work, I'm sure there are other cool things we can do here but good enough for now.